Common/UnitDefinitions/Luminance.json

```diff
@@ -0,0 +1,67 @@
+{
+  "Name": "Luminance",
+  "BaseUnit": "CandelaPerSquareMeter",
+  "BaseType": "double",
+  "XmlDoc": "Luminance is a photometric measure of the luminous intensity per unit area of light travelling in a given direction.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Luminance",
+  "BaseDimensions": {
+    "J": 1,
+    "L": -2
+  },
+  "Units": [
+    {
+      "SingularName": "CandelaPerSquareMeter",
+      "PluralName": "CandelasPerSquareMeter",
+      "FromUnitToBaseFunc": "{x}",
+      "FromBaseToUnitFunc": "{x}",
+	   "BaseUnits": {
+        "J": "Candela",
+        "L": "Meter"
+      },
+	  "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Cd/m²" ]
+        }
+      ]
+    },
+	{
+      "SingularName": "CandelaPerSquareFoot",
+      "PluralName": "CandelasPerSquareFoot",
+      "FromUnitToBaseFunc": "{x}* 1.07639e1",
+      "FromBaseToUnitFunc": "{x}/ 1.07639e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Cd/ft²" ]
+        }
+      ]
+    },
+	{
+      "SingularName": "CandelaPerSquareInch",
+      "PluralName": "CandelasPerSquareInch",
+      "FromUnitToBaseFunc": "{x}* 1.5500031e3",
+      "FromBaseToUnitFunc": "{x}/ 1.5500031e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Cd/s·in²" ]
```

Comment:
Should this not be `Cd/in²`?

Reply:
Thanks for cathing, yes it was a typo by going from sq-in to in². 
Additionally I realized that UnitEnumValues.g.json file is not overwritten and keeping some extra enums that is initially generated, I tried to delete this file totally and regenerate it by generate code but it seems to have a larger footprint so I avoided. What would you suggest to clear these extra fields?

Reply:
Ah, I did not spot these myself.

Try deleting everything under `"Luminance": { ... }` in the JSON file, then re-run codegen.
It should then allocate unit enum values only for the units that actually exist.

As you noticed, by design it does not remove any values if units are removed, to preserve backwards compatibility.

Reply:
I see, yes it makes sense. Cleaned that part, also thanks to @tmilnthorp for pointing out autogenerated parts as well.